packages/puppeteer-extra-plugin-devtools/package.json

```diff
@@ -7,10 +7,10 @@
   "author": "berstend",
   "license": "MIT",
   "scripts": {
-    "docs": "exit 0",
-    "test": "eslint --ext .js .",
-    "test-old": "ava -v && eslint --ext .js .",
-    "test-ci-old": "yarn test"
+    "docs": "node -e 0",
+    "lint": "eslint --ext .js .",
+    "test": "run-s lint",
```

Comment:
Did we accidentally drop `ava` code testing here? I don't mind splitting `lint` into it's own script, could still make sense to run it as well when `test` is called

Reply:
Oops, got eye! I put it back.